jwst/extract_1d/extract.py

```diff
@@ -3851,6 +3870,10 @@ def create_extraction(extract_ref_dict,
         spec.slit_dec = dec
         spec.spectral_order = sp_order
         spec.dispersion_direction = extract_params['dispaxis']
+        spec.xstart = extraction_values['xstart']
```

Comment:
When reviewing and merging the stdatamodels PR to add these new keyword entries to the `MultiSpecModel` I decided to make the names of the data model attributes a bit more unique and changed them to ``extraction_xstart``, ``extraction_xstop``, etc. So you'll need to update these 4 lines of code to use the new names.